WEBROOOOT/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	
+	<link rel="stylesheet" href="style.css">
+
+	 <title>ABOUT ME</title>
+
+
+	 <nav> 
+
+		<a href="index.html">(It's All) ALL ABOUT ME</a> 
+	 	<a href="portfolio.html">PORTF0LIO</a> 
+
+
+	 </nav>
+
+</center>
+
+</head>
+
+
+
+<body>
+
+
+    <h1> ALEXANRA DE LEON </h1>
+<br> 
+<br>
+<br>
+<center> <img src="FLOATINGPLANTE.gif">
```

Comment:
Very excited that you are finding new tags to use!  

There are a bunch of tags out there - including ```<center>``` that are deprecated - meaning that they existed in previous versions of HTML but are no longer officially part of HTML 5 (the latest version).  

W3Schools does a great job at listing out all of these tags and letting you know if one should no longer be used.  http://www.w3schools.com/tags/tag_center.asp